homeassistant/components/mqtt/__init__.py

```diff
@@ -310,6 +310,11 @@ def async_setup(hass, config):
         certificate = os.path.join(os.path.dirname(__file__),
                                    'addtrustexternalcaroot.crt')
 
+    # When the port indicates mqtts, use bundled certificates from requests
+    if certificate is None and port == 8883:
+        import requests.certs
```

Comment:
Fixed in https://github.com/home-assistant/home-assistant/pull/6429/commits/f543bb8e616a8da25582b89bd177227ed25bcd2e